### ISSUE TITLE
Add user controls for search weights

### ISSUE THREAD

**Issue by flibbertigibbet** [azavea/echo-locator#122]
Add drop-down inputs to the profile edit page for user preferences for weighting neighborhood variables by violent crime, school quality, and transit accessibility. Store the selections to the user profile. Handle loading outdated user profiles missing these new values by using defaults.

This task is only for adding the UI components, and not for modifying the ranking to use these values from the user profile.

Options should be a range of very important, important, and not important, with possibly a fourth option. All options should have defaults: 50% for transit travel time, 25% for schools, and 25% for crime.

~The weights should sum to 100% for the form to be valid. The simplest way to enforce this may be making the transit accessibility weight control read-only, and update it on change to either of the other two controls.~ Weights will have to be relatively dependent (e.g., ranking everything "unimportant" would weight the same as ranking everything "very important").

**Comment by lederer** [comment]
> The weights should sum to 100% for the form to be valid.

Could we try something like:

- Each of the 3 factors has a dropdown with [Very important, Important, Not important], set to appropriate defaults.
- User can set each option however they like. Setting one does not affect the value of another (this is preferable for a simple UX).
- Behind the scenes, each value amounts to a simple scalar for its dimension, eg, `Very important: 2`, `Important: 1`, `Not important: 0`.

**Comment by flibbertigibbet** [comment]
> * User can set each option however they like. Setting one does not affect the value of another (this is preferable for a simple UX).

The consequence of this would be that any weights settings that do not sum to 100% would result in an invalid form, and prevent the user from saving changes. Particularly since the labels do not indicate the corresponding percentage, this would be confusing.
